tests/foreman/cli/test_variables.py

```diff
@@ -283,7 +288,8 @@ def test_positive_update_variable_puppet_class(self):
         smart_variable = make_smart_variable({
             'puppet-class': self.puppet['name']})
         self.assertEqual(smart_variable['puppet-class'], self.puppet['name'])
-        new_puppet = Puppet.info({u'name': 'ntp::config'})
+        new_puppet = Puppet.info(
+            {u'name': self.puppet_modules[1]['name']})
```

Comment:
So you have predefined list of puppet modules on the top of the module, and then using unique items in tests by passing different ids, right? Well, for 2 tests it will work, but in case we increase tests amount it will be hard to manage unique ids this way. I'd rather use `.pop()` to not care which id was already used an by what specific test.